doc/UsersGuide/source/BuildingAndRunning.rst

```diff
@@ -187,13 +177,243 @@ The WM can be built by running the following command from the `ufs-weather-model
+``APP=S2S``), CCPP suite to use (e.g. ``SUITES=FV3_GFS_2017_coupled``), and
+additional build options (e.g. ``DEBUG=Y``) as necessary. The ``RUN`` line
+specifies the name of a test to run. The test name should match the name of one
+of the test files in the tests/tests/ directory or, if the user is adding a new
+test, the name of the new test file. The order of lines in ``rt.conf`` matters
+since ``rt.sh`` processes them sequentially; a ``RUN`` line should be proceeded
+by a ``COMPILE`` line that builds the model used in the test. The following example
+``rt.conf`` file builds the standalone ATM model in 32 bit and then runs the
+``control`` test:
+
+.. code-block:: console
+
+    COMPILE | APP=ATM SUITES=FV3_GFS_v16 32BIT=Y | | fv3
+    RUN     | control                            | | fv3
+
+The third column of ``rt.conf`` relates to the platform; if left blank, the test
+runs on all Tier-1 platforms. The fourth column deals with baseline creation (more
+on this later) and ``fv3`` means the test will be included during baseline creation.
+The ``rt.conf`` file includes a large number of tests. If the user wants to run
+only a specific test, s/he can either comment out (using the ``#`` prefix) the
+tests to be skipped, or create a new file, e.g. ``my_rt.conf``, then execute
+``./rt.sh -l my_rt.conf``.
+
+The regression test generates a number of log files. The summary log file
+``RegressionTests_<machine>.<compiler>.log`` in the tests/ directory compares
+the results of the test against the baseline specific to a given platform and
+reports the outcome (hence, the 'regression' test): 'Missing file' results when
+the expected files from the simulation are not found, and typically occurs
+when the simulation did not run to completion; 'OK' means that the simulation
+results are bit-for-bit identical to those of the baseline; 'NOT OK' when
+the results are not bit-for-bit identical; and 'Missing baseline' when there
+is no baseline data to compare against.
+
+More detailed log files are found in the tests/log_<machine>.<compiler>/ directory.
+In particular, the user may find useful the run directory path provided as the
+value of ``RUNDIR`` variable in the ``run_<test-name>`` file. ``$RUNDIR`` is a
+self-contained (i.e. sandboxed) directory with the executable file, initial
+conditions, model configuration files, environment setup scripts and a batch job
+submission script. The user can run the test by cd'ing into ``$RUNDIR`` and
+invoking the command
+
+.. code-block:: console
+
+    sbatch job_card
+
+This can be particularly useful for debugging and testing code changes. Note that
+``$RUNDIR`` is automatically deleted at the end of a successful regression test;
+specifying the ``-k`` option retains the ``$RUNDIR``, e.g. ``./rt.sh -l rt.conf -k``.
+
+Found inside the ``$RUNDIR`` directory are the model configuration files
+``ice_in``, ``input.nml``, ``model_configure`` and ``nems.configure``.
```

Comment:
perhaps we should say that input.nml, model_configure and nems.configure will always be there but that when running other configurations (eg s2s,s2sw,ng-godas) other files configuration files might also be present.

Reply:
I will add that explanation.